src/libs/Console/index.ts

```diff
@@ -118,5 +119,29 @@ function createLog(text: string) {
     }
 }
 
-export {sanitizeConsoleInput, createLog, shouldAttachLog};
+/**
+ * Loops through all the logs and parses the message if it's a stringified JSON
+ * @param logs Logs captured on the current device
+ * @returns CapturedLogs with parsed messages
+ */
+function parseStringifyMessages(logs: Log[]): Log[] {
```

Comment:
Perhaps `Stringified` instead?
```suggestion
function parseStringifiedMessages(logs: Log[]): Log[] {
```